docs/index.rst

```diff
@@ -16,6 +16,17 @@ them back periodically. Typically these external sources are version control
 repositories that store the strings for an application. Supported external
 sources include **Git**, **Mercurial** and **Subversion**.
 
+Other Documentation
+-------------------
+
+Important additional Pontoon documentation can also be found at the following sites:
+
+`How To Use Pontoon <https://mozilla-l10n.github.io/localizer-documentation/tools/pontoon/>`_ (Mozilla Localizer Documentation)
+
+`Working with Pontoon <https://mozilla-l10n.github.io/documentation/tools/pontoon>`_ (Mozilla Internal Tools Documentation)
```

Comment:
To make the distinction between the two docs clearer, I'd rename the link text:
- How To Use Pontoon  => How to localize with Pontoon
- Working with Pontoon => How to manage localizations on Pontoon